src/test/java/com/revature/modelTests/TechnologyTests.java

```diff
@@ -0,0 +1,8 @@
+package com.revature.modelTests;
+
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+public class TechnologyTests {
```

Comment:
Must add tests to this class that test getters/setters/hashcode/equals of `Technology` class.  

You can hard code them or try adding Bean Matchers for a short cut. Resource [here](https://github.com/orien/bean-matchers)